Makefile

```diff
@@ -47,7 +47,7 @@ sources: tarball specfile
 
 .PHONY: rhel8-support
 rhel8-support:
```

Comment:
Although I think rhel8-support should also be changed to something like shebang-support, you can leave it as it is for now.

Reply:
I went ahead and renamed it.  See commit 0c9a274.